.eslintrc.json

```diff
@@ -21,6 +21,10 @@
             "ClassDeclaration": false
         }
     }],
+    "prefer-const": ["error", {
+        "destructuring": "any",
+        "ignoreReadBeforeAssign": false
+    }],
```

Comment:
yep definitely, this comes up all the time in code review for me https://github.com/artsy/emission/pull/380#pullrequestreview-5433438